Hai Reveal conversations: reduce text walls in missions A12 & A13 (#7593)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

			label continue
			`	When you finish on the possibility of an Unfettered war fleet appearing at the wormhole to scour pirate worlds, he stops you with a pained expression. "Well, it's all falling apart, isn't it... Alright, look, this is above my pay-grade, and I don't think within Terry's either. You need to go contact Admiral Danforth. The connection from here to Farpoint is excellent, now since all our traffic is routing through those new relays, so it should be easy enough. I'll make sure no one does anything stupid from this side. I'm sure there's a solution here, we all just need to keep our heads and call the right people."`
			choice

> 				`	"I'll go to my ships comms now and call him."`

Reviewer: ```suggestion
				`	"I'll go to my ship's comms now and call him."`
```

	on offer
#		log ``
		conversation

> 			`You are granted access to the secure spaceport, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day and there is a gloom developing as the sun disappears behind the buildings of the capital when you enter the diplomatic wing of the government complex. Fortunately, though the place appears nearly deserted, you swiftly run into Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey.`

Reviewer: ```suggestion
			`You are granted access to the secure spaceport, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day, and there is a gloom developing as Ya Hai disappears behind the buildings of the capital, when you enter the diplomatic wing of the government complex. Fortunately, though the place appears nearly deserted, you swiftly run into Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey.`
```

Reviewer: Actually not too sure about this one.

			label summary
			`	The freshest of the group, Batu, seems to twig to something first. "There's going to be an ultimatum, isn't there? The fleet is a threat."`
			`	"Yes," says Sayari. "It's going to be a very broad ultimatum though, with lots of room for the Republic to decide how it's addressed; and no one is shooting at anyone without talking first. Now, all of you except <last> are traveling with Elder Teeneep. I suggest you go and prepare immediately."`
			`	As you begin to walk away, Sayari motions for you. It seems she wants a private discussion.`
			choice
				`	(Wait for the others to leave.)`

> 			`	After the others are gone, Sayari explains the situation. "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change elsewhere who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`

Reviewer: ```suggestion
			`	After the others are gone, Sayari explains the situation. "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change, who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`
```

			`	He gesticulates, apparently at a loss for words. "Who even wrote this? Who writes this stuff? Why would you even put that in there?"`
			`	"Sorry, you were presumably here on business, what's going on out there?"`
			label interrupt
			choice
				`	"The Hai have closed their borders and may shoot anyone trying to pass through the wormhole."`
					goto closed
				`	"The Hai are having an election, and Sayari is expecting political chaos. Is everyone here okay?"`
			`	"They're having a what?" He listens incredulously as you explain the situation with pirate slavers, the Unfettered support, the public shaming and the snap election.`
				goto continue
			label closed
			`	"We suspected that, we haven't been game to test, but why?" He listens incredulously as you explain the situation with pirate slavers, the Unfettered support, the public shaming and the snap election.`
			label continue

> 			`	When you finish on the possibility of an Unfettered war fleet appearing at the wormhole to scour pirate worlds, he stops you with a pained expression. "Well, it's all falling apart, isn't it... Alright, look, this is above my pay-grade, and I don't think within Terry's either. You need to go contact Admiral Danforth. The connection from here to Farpoint is excellent, now since all our traffic is routing through those new relays, so it should be easy enough. I'll make sure no one does anything stupid from this side. I'm sure there's a solution here, we all just need to keep our heads and call the right people."`

Reviewer 2: ```suggestion
			`	When you finish on the possibility of an Unfettered war fleet appearing at the wormhole to scour pirate worlds, he stops you with a pained expression. "Well, it's all falling apart, isn't it... Alright, look, this is above my pay-grade, and I don't think within Terry's either. You need to go contact Admiral Danforth. The connection from here to Farpoint is excellent now, since all our other traffic is routing through those new relays, so it should be easy enough. I'll make sure no one does anything stupid from this side. I'm sure there's a solution here, we all just need to keep our heads and call the right people."`
```
My bad

			label summary
			`	The freshest of the group, Batu, seems to twig to something first. "There's going to be an ultimatum, isn't there? The fleet is a threat."`
			`	"Yes," says Sayari. "It's going to be a very broad ultimatum though, with lots of room for the Republic to decide how it's addressed; and no one is shooting at anyone without talking first. Now, all of you except <last> are traveling with Elder Teeneep. I suggest you go and prepare immediately."`
			`	As you begin to walk away, Sayari motions for you. She leads you away to have a private discussion.`
			choice

> 				`	"What is it, Sayari?"`

Reviewer 3: Having another choice here might be good.
```suggestion
				`	"What is it, Sayari?"`
				`	(Follow her.)`
```

Author: You would already have followed her by the time you ask, "What is it, Sayari?"

I do need a second option, though. I will ponder it. Either that, or I'll put the paragraph back together.

Author: Okay. There are two options now. How do you like them?

	on offer
#		log ``
		conversation

> 			`You are granted access to the secure spaceport, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day, and there is a gloom developing as Ya Hai disappears behind the buildings of the capital, when you enter the diplomatic wing of the government complex. Fortunately, though the place appears nearly deserted, you swiftly run into Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey.`
> 			`	When he sees you Alondo raises a hand and gestures for you to sit with them. "<first>! Bloody hell, come join us."`

Reviewer 2: I do think this bit of clarity is relevant, since you'd expect to be blocked right now. Otherwise the elegance of these edits got me thinking on a slightly different path about how to word this though so I hope this sounds good. :)
```suggestion
			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day and the developing gloom as Ya Hai sinks behind the buildings of the capital underscores the mood when you enter the diplomatic wing of the government complex. The place appears nearly deserted.`
			`	Fortunately though, you swiftly discover Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey. When he sees you Alondo raises a hand and gestures for you to sit with them. "<first>! Bloody hell, come join us."`
```

			`	Alondo grimaces, raises his eyebrows, and gives a lopsided shrug which altogether does a remarkable job of conveying the sentiment: 'so far it looks like it, but I still need at least one more stiff drink to get over it'. Remington looks at him and chuckles halfheartedly before emptying his glass and putting it down with a sharp clink. "We weren't sure for a while there, but other than being cut off, we're okay."`
			label continue
			`	Alondo gestures at the large, wall-mounted monitor they've been watching, and you realize that it must be displaying some kind of post-election coverage. The writing is all in Hai, but it seems that the majority of the information that would matter to the diplomats has been displayed visually. On its own you'd have never guessed at trying to read Teeneep's name written in Hai, but when it's presented next to her face and then shown on a pie chart next to percentages and other graphics, it becomes easy enough to follow what's happening.`
			choice
				`	"Who won the election?"`
					goto "who won"

> 				`	"Can you explain all of this, Alondo?"`

Reviewer 2: ```suggestion
				`	"Can either of you explain all of this?"`
```

				goto "walk or drink"
			label "who won"
			`	Remington answers your question, "That Teeneep of yours, she had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot. She's a firebrand, that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech.`
			label "walk or drink"

> 			`	"Grab yourself a drink and join us." You notice a bartender off to your left.`

Reviewer 2: They are intentionally alone, abandoned by the Hai who are dealing with their own concerns. They've retrieved a bottle from somewhere and you'd have to get yourself a glass, though you can probably work out how to pour yourself water from behind the bar easily enough too.

Author: I changed it so Alondo offers to get you a drink.

			label summary
			`	The freshest of the group, Batu, seems to twig to something first. "There's going to be an ultimatum, isn't there? The fleet is a threat."`
			`	"Yes," says Sayari. "It's going to be a very broad ultimatum though, with lots of room for the Republic to decide how it's addressed; and no one is shooting at anyone without talking first. Now, all of you except <last> are traveling with Elder Teeneep. I suggest you go and prepare immediately."`
			`	As the meeting disperses, Sayari motions for you. She leads you away to have a private discussion.`
			choice
				`	"What has Teeneep done, Sayari?"`
					goto hasdone
				`	"I take it you need another ride?"`
			`	She puts her paw over her face and breathes a long sigh. "Yes, Captain <last>, I do need another ride." She smiles and laughs a little before looking at you again.`
			label hasdone

> 			`	Sayari explains the situation. "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change, who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`

Reviewer 2: ```suggestion
			`	She explains, "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change, who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`
```

	on offer
#		log ``
		conversation

> 			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day and the developing gloom as Ya Hai sinks behind the buildings of the capital underscores the mood when you enter the diplomatic wing of the government complex. The place appears nearly deserted.`

Reviewer: ```suggestion
			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day when you enter the diplomatic wing of the government complex. The developing gloom as Ya Hai sinks behind the buildings of the capital underscores the mood; the place appears nearly deserted.`
```
This sentence is way too long again.

			`	He manages to make that last line sound both bewildered and grumpy at the same time. "That Teeneep you've been running around with - and yes, don't look at me in shock, Sayari told me when we were concerned you might not get out. You've developed some friends in high places and I admire that... where was I?"`
			`	"Teeneep!" says Alondo, perhaps a little too loudly, nearly poking Remington in the face.`
			`	"Of course, that Teeneep of yours, she had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot, at least now that 'sir pitchforks and pulse rifles' down there has definitively lost. She's a firebrand, that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech.`
				goto "walk or drink"
			label "who won"

> 			`	Remington answers your question, "That Teeneep of yours, she had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot. She's a firebrand, that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech.`

Reviewer: ```suggestion
			`	Remington answers your question. "That Teeneep of yours, she had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot. She's a firebrand, that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech.`
```
That is all, I think.

			label "walk or drink"

> 			`	Alondo walks over to the bar and says, "I'll get you a drink, <last>. What'll you have?"`

Reviewer 2: ```suggestion
			`	Alondo wobbles over to the bar and says, "I'll get you a drink, <last>. What'll you have?"`
```
he ain't sober
